wdl/ExpansionHunterScatter.wdl

```diff
@@ -0,0 +1,47 @@
+version 1.0
+
+import "Structs.wdl"
+import "ExpansionHunter.wdl" as ExpansionHunter
+
+workflow ExpansionHunterScatter {
+
+    input {
+        Array[File] bams_or_crams
```

Comment:
Regarding output prefixes: often bam/cram filenames can be difficult to interpret at a glance. I would suggest adding an optional input `Array[String]? sample_ids` that are used as the output prefix for each sample, if provided. This can also guarantee that, within a given cohort, the filenames are unique, which can be helpful for managing data downstream.

Reply:
That would be very useful, thank you. I added the optional list.